webhooks/mutatepod_parts.go

```diff
@@ -42,7 +42,7 @@ func k8sSATokenVolume(
 						Path:              K8sSATokenName,
 					},
 				}},
-				DefaultMode: pointer.Int32(0440),
+				DefaultMode: pointer.Int32(444),
```

Comment:
`0444`(octal) would be correct instead of `444`(decimal).

```suggestion
				DefaultMode: pointer.Int32(0444),
```